build.gradle

```diff
@@ -0,0 +1,60 @@
+buildscript {
+    repositories {
+        mavenCentral()
+        mavenLocal()
+        maven {
+            url 'https://plugins.gradle.org/m2/'
+        }
+    }
+    dependencies {
+        classpath 'com.cinnober.gradle:semver-git:2.2.1'
+        classpath 'net.ltgt.gradle:gradle-errorprone-plugin:0.0.8'
+    }
+}
+
+
+
+allprojects {
+    apply plugin: 'com.cinnober.gradle.semver-git'
+    apply plugin: 'idea'
+    apply plugin: 'eclipse'
+    apply plugin: 'java'
+    repositories {
+        maven {
+            url "http://maven.fraudmetrix.cn/nexus/content/groups/public/"
+        }
+        mavenLocal()
+        mavenCentral()
+
+
+    }
+    rootProject.version='3.0.0'
+    version = rootProject.version
+
+
+    plugins.withType(JavaPlugin) {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        dependencies {
+            testCompile('junit:junit:4.12')
+            compile('log4j:log4j:1.2.16')
+            compile('org.codehaus.jackson:jackson-core-asl:1.9.5')
+            compile('org.codehaus.jackson:jackson-mapper-asl:1.9.5')
+            compile('org.apache.hadoop:hadoop-common:2.7.2')
+            compile('azkaban:azkaban-common:3.0.0')
```

Comment:
i also found some jar which can not be found in public repository,like teradata-connector-1.4.2.jar. how we manager  those jars?